samples/wifi/scan/README.rst

```diff
@@ -20,9 +20,30 @@ Overview
 ********
 
 This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.
+The default scan type is active scan.
+However, you can configure the scan type to be either active or passive.
 
 Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.
 
+Configuration
+*************
+
+|config|
+
+Select the scan type to be used by the sample:
```

Comment:
```suggestion
Select the scan type to be used by the sample.
```
Alternatively:
Now, if there are no other important configuration options, you could dump the Configuration options section and just have it in this Configuration section.
Something like: "Use the :kconfig:option:`CONFIG_WIFI_SCAN_TYPE_SELECT` Kconfig option to set the scan type to be used by the sample."